student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown')
```

Comment:
- [ ] This constructor is meant to have all the parameters of Person class too and by inclusion, classroom. I will encourage you to add the parent's permission too.

![image](https://github.com/clin2on3mun/OOP-School-Library/assets/86200356/0ffc36df-0161-4c7a-8361-f4d3f8d8aa5c)


Remember to include it in the super too.